instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/context/InstrumentationApiContextBridging.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opentelemetryapi.context;
+
+import application.io.opentelemetry.api.trace.Span;
+import io.opentelemetry.javaagent.instrumentation.opentelemetryapi.trace.Bridging;
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import javax.annotation.Nullable;
+
+@SuppressWarnings({"unchecked", "rawtypes"})
+final class InstrumentationApiContextBridging {
+
+  static List<ContextKeyBridge<?, ?>> instrumentationApiBridges() {
+    List<ContextKeyBridge<?, ?>> bridges = new ArrayList<>();
+
+    for (String spanKeyName :
+        Arrays.asList(
+            "SERVER_KEY",
+            "HTTP_CLIENT_KEY",
+            "RPC_CLIENT_KEY",
+            "DB_CLIENT_KEY",
+            "CLIENT_KEY",
+            "PRODUCER_KEY",
+            "CONSUMER_RECEIVE_KEY",
+            "CONSUMER_PROCESS_KEY")) {
+      ContextKeyBridge<?, ?> spanKeyBridge = spanKeyBridge(spanKeyName);
+      if (spanKeyBridge != null) {
+        bridges.add(spanKeyBridge);
+      }
+    }
+
+    ContextKeyBridge<?, ?> httpRouteHolderBridge = httpRouteHolderBridge();
+    if (httpRouteHolderBridge != null) {
+      bridges.add(httpRouteHolderBridge);
```

Comment:
and if I remove the http route bridge here, the tests still pass

Reply:
Right, the test wasn't properly constructed... fixed that.
Thanks!